themes/godotengine/pages/education.htm

```diff
@@ -0,0 +1,82 @@
+title = "Education"
+url = "/education"
+layout = "default"
+is_hidden = 0
+==
+{##}
+{% put head_text %}
+<p class="small">
+  Godot is already being used in many classrooms across the world. If you are interested in including it in your
+  curriculum, you can find answers to some of the most common questions on this page.
+</p>
+{% endput %}
+
+<style>
+  .question h3 {
+    margin: 2em 0 0;
+  }
+
+  .question p {
+    margin-top: 0.5em;
+  }
+</style>
+
+<div class="question">
+  <h3>Can I teach Godot in my classroom?</h3>
+  <p>Godot&nbsp;Engine is <a href="https://en.wikipedia.org/wiki/Free_and_open_source_software">free and open source
+      software</a> released under the permissive <a href="/license">MIT license</a>.<br>
+    You can use Godot in your classroom without any restrictions and with zero licensing
+    costs.
+  </p>
+  <p>You can read more about <a href="/license">our license here</a>.</p>
+</div>
+
+<div class="question">
+  <h3>Will the Godot project sign a services agreement provided by my institution?</h3>
+  <p>The Godot Engine project is unable to accept any terms and conditions imposed by organizations, nor will it
+    provide warranties or representations about the Godot Engine. This is clearly outlined in the text of the <a
+      href="/license">MIT license</a>.</p>
+  <p>As a free and open source project, Godot is not backed by a corporation or any individual. Accordingly, it doesn't
+    have the legal capacity to enter into agreements, even if such agreements did not conflict with the MIT license.</p>
+</div>
+
+<div class="question">
+  <h3>What data does the Godot Engine executable collect from users?</h3>
+  <p>When you use the Asset Library in the Godot Engine application, HTTPS requests are made to the Asset Library
+    website, and will contain: Godot Engine version, search query, selected assets, and the user's IP address. The Asset
+    Library
+    processes this data without storing it, but our TuxFamily hosting's HTTP server may log these requests. Such logs
+    are subject to TuxFamily's privacy policy (https://faq.tuxfamily.org/Privacy/En).<br>
+    This is the only personal data collected by the Godot Engine application.</p>
+  <p>To be clear, the Godot Engine executable does not ask for or otherwise collect personally identifying information
+    from
+    its users. The HTTPS requests include the bare minimum amount of information required to do an online search and
+    transmit substantially less information than most internet browsing will.</p>
```

Comment:
I'm not sure if we could not scratch "HTTPS" from this, because its an unnecessary technical detail and may change (even if its unlikely to). It could even change depending on local network setup (i.e., proxies/SSL stripping).

Maybe just mention "network requests", that encryption is used for protection during transmission, and maybe mention that european data protection/GDPR is adhered to and the servers are located in europe, not sure. From what I've seen, european institutions may especially want to know the latter.

Sidenote, but "server logging" and "IPs" are a dicey subject, at least in germany (no idea about france, but both are EU at least and GDPR applies). Not a lawyer, but from what I've seen, its still up in the air whether IPs are considered personal data (trends to yes; depends on court), and logging may not always be legal without explicit user consent even if you argue its technical necessity to provide the service (more explicit consent at least than "its in our EULA"/you agree by downloading). I've seen companies make sure after GDPR to either only log pseudonymous IPs (i.e., the last bits zeroed out) or only keep them in very short lived logs for debugging (24 hours, maybe). We may want to check/confirm here.

Reply:
> I'm not sure if we could not scratch "HTTPS" from this, because its an unnecessary technical detail and may change (even if its unlikely to). It could even change depending on local network setup (i.e., proxies/SSL stripping).
> 
> Maybe just mention "network requests", that encryption is used for protection during transmission, and maybe mention that european data protection/GDPR is adhered to and the servers are located in europe, not sure. From what I've seen, european institutions may especially want to know the latter.

I agree, commented above with suggested changes

> Sidenote, but "server logging" and "IPs" are a dicey subject, at least in germany (no idea about france where the servers presumably are located, but both are EU at least and GDPR applies). Not a lawyer, but from what I've seen, its still up in the air whether IPs are considered personal data (trends to yes; depends on court), and logging may not always be legal without explicit user consent even under the "technical necessity to provide the service" caveat (may require more explicit consent at least than "its in our EULA"/you agree by downloading). I've seen companies make sure after GDPR to either only log pseudonymous IPs (i.e., the last bits zeroed out) or only keep them in very short lived logs for debugging (24-48 hours?). We may want to check/confirm here.

I'm worried this may be a little out of scope for this document. It sounds like we may want to investigate a little deeper into what the impact of TuxFamily's server logging may be. I suggest that we come back to this point later

> Notably, we also have https://godotengine.org/privacy-policy which seems relevant, maybe we should link to that?

Yes, we should add a link to the privacy policy here